libs/shared/auth/src/lib/user-info/user-info.component.ts

```diff
@@ -13,6 +13,6 @@ export class UserInfoComponent {
   @Input() isLoggedIn: boolean | null = false;
   @Input() userProfile: KeycloakProfile | null = null;
 
-  @Output() login = new EventEmitter<any>();
-  @Output() logout = new EventEmitter<any>();
+  @Output() login = new EventEmitter<void>();
```

Comment:
were there compile errors?

Reply:
There were not. I just thought that void is more representative of the fact that the emitter carries no payload